modules/network/smb2man/libsmb2/lib/asn1-ber.c

```
@@ -0,0 +1,1078 @@
+    {
+        result = asn1ber_out_byte(actx, (const uint8_t)(val >> (8 * (bytesneeded - 1))));
+        if (result)
+        {
+            return result;
+        }
+        bytesneeded--;
+    }
+    return result;
+}
+
+int asn1ber_ber_from_uint64(struct asn1ber_context *actx, const ber_type_t type, const uint64_t val)
```

Cppcheck (2.21.0)

[style] The function 'asn1ber_ber_from_uint64' is never used.

(unusedFunction)